src/components/nav-bar/NavBar.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { PageHeader, Typography, Button, Avatar, Menu, Dropdown } from 'antd';
+import { UserOutlined } from '@ant-design/icons';
+import './nav-bar.less';
+const { Paragraph } = Typography;
+
+const NavBar: React.FC = () => {
+  const history = useHistory();
+  const location = useLocation();
+  const path = location.pathname;
+
+  const isLoggedIn = () => {
+    return !(path === '/' || path === '/login' || path === 'signup');
+  };
+
+  const BackIcon = () => {
+    const Logo: string = require('../../nav-bar-icon.png');
+    return <img className="back-icon" src={Logo} alt="icon" />;
+  };
```

Comment:
This seems a bit excessive to get a back icon. What's the icon and how come we don't just use one from AntD or an SVG?

Reply:
I can't get this to work without doing this, and it needs to be this cause it's a SFTT logo really not a back icon